src/network/network.hpp

```diff
@@ -17,7 +17,7 @@
 #include "config/config.hpp"
 #include "consensus/vote.hpp"
 #include "dag/dag_block_manager.hpp"
-#include "taraxa_capability.hpp"
+#include "tarcap/new_taraxa_capability.hpp"
```

Comment:
do not forget to rename it. or are we planning to call it new?

Reply:
no I will rename it together with cleaning old network files in single commit